testing/ncids-css-testing/stories/uswds-native/usa-step-indicator/src/index.scss

```diff
@@ -0,0 +1,3 @@
+@use "styles/ncids";
+@forward "usa-step-indicator";
+@forward "uswds-helpers";
```

Comment:
oh my bad. uswds-global also imports uswds-helpers. sorry, should have caught that at my last look over